gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -120,6 +121,7 @@
   private final Observable<SaveResult> saveResults;
 
   @Nullable private String fieldWaitingForPhoto;
+  @Nullable private String capturedPhotoPath;
```

Comment:
Why is this needed in the view model and stored across re-creation events?

Reply:
For photo capture with full size, we don't get a URI. While creating the intent, we generate and pass the file URI. The result of photo capture is of type boolean. `true` refers to that the photo was saved to the passed uri correctly.

Reply:
So we don't get the URI back from the intent, but we need it in order to serialize? Is it the URL of the full-sized image in local storage? Please add a comment here to clarify if possible.

Reply:
Done